packages/libirmin/libirmin.3.6.0/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+synopsis: "C bindings for irmin"
+description: "C bindings for irmin using Ctypes inverted stubs"
+maintainer: ["zachshipko@gmail.com"]
+authors: ["Zach Shipko"]
+license: "ISC"
+homepage: "https://github.com/mirage/irmin"
+bug-reports: "https://github.com/mirage/irmin/issues"
+depends: [
+  "dune" {>= "2.9"}
+  "ctypes" {>= "0.19"}
+  "ctypes-foreign" {>= "0.18"}
+  "irmin" {= version}
+  "irmin-cli" {= version}
+]
+build: [
+  ["dune" "subst"] {dev}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
```

Comment:
```suggestion
  ["dune" "runtest" "-p" name "-j" jobs] {with-test & os != "macos"}
```
```
#=== ERROR while compiling libirmin.3.6.0 =====================================#
# context              2.1.4 | macos/arm64 | ocaml-base-compiler.4.14.1 | pinned(https://github.com/mirage/irmin/releases/download/3.6.0/irmin-3.6.0.tbz)
# path                 ~/.opam/4.14.1/.opam-switch/build/libirmin.3.6.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p libirmin -j 7
# exit-code            1
# env-file             ~/.opam/log/libirmin-19509-a67695.env
# output-file          ~/.opam/log/libirmin-19509-a67695.out
### output ###
# File "src/libirmin/lib/dune", line 1, characters 0-115:
# 1 | (rule
# 2 |  (targets irmin_bindings.ml irmin.c irmin.h)
# 3 |  (deps
# 4 |   (:gen ../gen/generate.exe))
# 5 |  (action
# 6 |   (run %{gen} .)))
# (cd _build/default/src/libirmin/lib && ../gen/generate.exe .)
# Fatal error: exception Failure("ca-certs: empty trust anchors.\nPlease report an issue at https://github.com/mirage/ca-certs, including:\n- the output of uname -s\n- the distribution you use\n- the location of default trust anchors (if known)\n")
```

Reply:
> opam CI (presumably because of [mirage/ca-certs#20](https://github.com/mirage/ca-certs/issues/20)).

It's not just opam-repo-ci. It's everyone installing this package using opam. Our CI is just the only one that actually use what users will use (i.e. the sandbox)

For some reason I misread the error message and thought it would only be the tests but indeed if the whole package fails, this package should be entirely marked unavailable on macOS.

If you want to restore the availability on macOS you need to either find a way to fix ca-certs or remove the use of ca-certs in your build.